dist/scss/60_components/_c-payment-methods.scss

```diff
@@ -0,0 +1,70 @@
+@import '../10_settings/__all';
+@import '../20_tools/__all';
+
+.c-payment-methods {
+  margin-top: $global-spacing-unit-large;
+
+  label {
+    font-size: .7rem;
+    font-weight: bold;
+  }
+}
+
+.c-payment-methods__header {
+  @include breakpoint(tablet) {
+    padding-left: 0;
+    padding-right: 0;
+    width: 95%;
+    margin-left: auto;
```

Comment:
margin: auto should do the job instead of margin-left: auto & margin-right: auto.